src/transport_picoquic.cpp

```diff
@@ -478,8 +478,8 @@ PicoQuicTransport::Start()
      *    also triggers PMTUD to run. This value will be the initial value.
      */
     picoquic_init_transport_parameters(&local_tp_options_, 1);
-    local_tp_options_.max_datagram_frame_size = 1280;
-    //  local_tp_options.max_packet_size = 1450;
+    // local_tp_options_.max_datagram_frame_size = 1280;
+    //   local_tp_options.max_packet_size = 1450;
```

Comment:
Dead code?

Reply:
It was more of a note, but I can remove it.

Reply:
+1 to rmeoving this